core/src/test/kotlin/api/RunsRouteIntegrationTest.kt

```diff
@@ -198,6 +208,52 @@ class RunsRouteIntegrationTest : AbstractIntegrationTest({
         checkLogArchive(downloadFile, sources)
     }
 
+    "GET /runs/{runId}" should {
+        "return the requested ORT run" {
+            integrationTestApplication {
+                val run = ortRunRepository.create(
+                    repositoryId,
+                    "revision",
+                    null,
+                    JobConfigurations(),
+                    "jobConfigContext",
+                    labelsMap
+                )
+
+                val response = superuserClient.get("/api/v1/runs/${run.id}")
+
+                response shouldHaveStatus HttpStatusCode.OK
+                response shouldHaveBody run.mapToApi(Jobs())
+            }
+        }
+
+        "include job details" {
+            integrationTestApplication {
+                val run = ortRunRepository.create(
+                    repositoryId,
+                    "revision",
+                    null,
+                    JobConfigurations(),
+                    "testContext",
+                    labelsMap
+                )
+
+                val response = superuserClient.get("/api/v1/runs/${run.id}")
+
+                response shouldHaveStatus HttpStatusCode.OK
+                response shouldHaveBody run.mapToApi(Jobs())
```

Comment:
This is basically the same test as the previous one. Where are job details tested?

Reply:
I just copied the existing test cases and didn't notice those tests were identical. I have added commits to address that.